Jenkinsfile

```diff
@@ -0,0 +1,20 @@
+#!groovy
+try {
+    node("ecs") {
+        stage("Checkout") {
+            checkout scm
+        }
+        stage("Install dependencies") {
+            sh("yarn install")
+        }
+        stage("Tests") {
+            sh("yarn test")
+        }
+        stage('Finish') {
+            echo("Done. KTKSBAI!")
```

Comment:
ahahah now I'm confused as well, urban dictionary states [this](http://www.urbandictionary.com/define.php?term=ktksbi), but I'm not sold 😂